velox/py/plan_builder/PyPlanBuilder.h

```diff
@@ -0,0 +1,146 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#include "velox/core/PlanNode.h"
+#include "velox/exec/tests/utils/PlanBuilder.h"
+#include "velox/py/type/PyType.h"
+
+namespace facebook::velox::py {
+
+/// Called from when the Python module is loaded/initialized to register all
+/// resources and structures needed by Velox.
+void registerAllResources();
+
+class PyVector;
+
+/// Wrapper class to expose Velox plan nodes to Python.
+class PyPlanNode {
+ public:
+  explicit PyPlanNode(core::PlanNodePtr planNode = nullptr)
+      : planNode_(std::move(planNode)) {}
+
+  core::PlanNodePtr getPlanNode() const {
```

Comment:
return const & ?

C++ coding style asks to drop 'get' from getters.

Reply:
I'll remove from the C++ wrapper API, but in Python-land people usually use the get_ and set_ prefixes - so I'll keep them there at least for now